src/experimental/information_estimates.cpp

```diff
@@ -0,0 +1,407 @@
+#include <math.h>
+#include "slam_toolbox/experimental/information_estimates.hpp"
+
+#include <iostream>
+
+InformationEstimates::InformationEstimates(kt_double sensor_range, kt_double resolution, kt_double lambda, kt_double nu)
+{
+    m_max_sensor_range = sensor_range;
+    m_cell_resol = resolution;
+    m_obs_lambda = lambda;
+    m_obs_nu = nu;
+
+    m_map_dist = 200.0f; 
+    m_num_cells = static_cast<int>(m_map_dist / m_cell_resol);
+    
+    m_curr_mut_info = 0.0;
+
+    m_mutual_grid.resize(m_num_cells, m_num_cells);
+    m_visited_grid.resize(m_num_cells, m_num_cells);
+}
+
+InformationEstimates::InformationEstimates()
+{
+    m_max_sensor_range = 5.0;
+    m_cell_resol = 0.05;
+    m_obs_lambda = 0.35; 
+    m_obs_nu = 0.28;
+
+    m_map_dist = 200.0f; 
+    m_num_cells = static_cast<int>(m_map_dist / m_cell_resol);
+
+    m_curr_mut_info = 0.0;
+
+    m_mutual_grid.resize(m_num_cells, m_num_cells);
+    m_visited_grid.resize(m_num_cells, m_num_cells);
+}
+
+std::tuple<int, kt_double> InformationEstimates::calculateMutualInformation(std::vector<karto::LocalizedRangeScan*> const& range_scans)
```

Comment:
@kmilo7204 hmm, consider changing the method name to reflect what the function actually does.